<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelCheckDialogController</class>
 <widget class="QDialog" name="ModelCheckDialogController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STD Configurations</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GUIResources.qrc">
    <normaloff>:/resources/resources/introduction1.png</normaloff>:/resources/resources/introduction1.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="labelModelFile">
      <property name="text">
       <string>Model File</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelPathToFile">
      <property name="text">
       <string>path_to_and_name_of_file</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditFormula">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>PR&lt;=0.2(X.reservoir&lt;=5.0)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboFormula">
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (X.reservoir &lt;= 5.0)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (M.pumpon = 0)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (NEG M.pumpon = 0)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (M.pumpon = 0 AND X.reservoir &lt;= 5.0)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (M.pumpon = 0 U[0,10] X.reservoir &lt;= 5.0)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (M.input1on = 1)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (X.soft1 &lt;= 0.1)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (X.soft1 &lt;= 0.2 AND X.soft1 &lt;= 0.1)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (NEG X.soft1 &lt;= 0.1)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (NEG M.input1on = 1)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (TT)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 (M.input1on=1 U[0,10] X.soft1&lt;=0.1)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PR&lt;=0.2 ((M.input1on=1 AND X.soft1&lt;=0.1) U[0,10] X.stor2&lt;=0.2)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelTTC">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Time To Check</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditTTC">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelMaxTime">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Maximum Time</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEditMaxTime">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="radioTextInput">
      <property name="text">
       <string>Textual Input</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelFormula">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Formula</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="radioDemoInput">
      <property name="text">
       <string>Demo Input</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioTextInput</tabstop>
  <tabstop>lineEditFormula</tabstop>
  <tabstop>radioDemoInput</tabstop>
  <tabstop>comboFormula</tabstop>
  <tabstop>lineEditTTC</tabstop>
  <tabstop>lineEditMaxTime</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="GUIResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModelCheckDialogController</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModelCheckDialogController</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditFormula</sender>
   <signal>textEdited(QString)</signal>
   <receiver>radioTextInput</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboFormula</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>radioDemoInput</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
